index.html

```diff
@@ -285,7 +285,7 @@ <h2 class="heading-2">Contact me</h2>
                             with it
                         </p>
                     </div>
-                    <form action="https://formspree.io/f/mgedjkyb" id="form1" method="post" class="contact-form">
+                    <form noValidate action="https://formspree.io/f/mgedjkyb" id="form1" method="post" class="contact-form">
```

Comment:
Great job team 🤝

- [ ] As specified in the project requirements, your form is supposed to submit if the validation is okay. Unfortunately, your form is not submitted even when the email is lowercase. Kindly make sure your form is submitted properly to `form spree`.